kw2xrf: linking problems with arm-none-eabi-gcc 7.1.0 and *-ld 2.28

With the latest arm-none-eabi-gcc (7.1.0) and -ld (2.28) on my Arch machine, I get the following linking problems when building `gnrc_networking` for the `pba-d-01-kw2x` board:
```
% BOARD=pba-d-01-kw2x make
RIOT/examples/gnrc_networking/bin/pba-d-01-kw2x/kw2xrf.a(kw2xrf.o): In function `kw2xrf_reset_phy':                                                                                                                                                                                     
RIOT/drivers/kw2xrf/kw2xrf.c:126: undefined reference to `kw2xrf_clear_dreg_bit'
RIOT/drivers/kw2xrf/kw2xrf.c:128: undefined reference to `kw2xrf_enable_irq_b'
RIOT/examples/gnrc_networking/bin/pba-d-01-kw2x/kw2xrf.a(kw2xrf_netdev.o): In function `_isr':
RIOT/drivers/kw2xrf/kw2xrf_netdev.c:694: undefined reference to `kw2xrf_mask_irq_b'
RIOT/drivers/kw2xrf/kw2xrf_netdev.c:754: undefined reference to `kw2xrf_enable_irq_b'
RIOT/examples/gnrc_networking/bin/pba-d-01-kw2x/kw2xrf.a(kw2xrf_getset.o): In function `kw2xrf_abort_sequence':
RIOT/drivers/kw2xrf/kw2xrf_getset.c:152: undefined reference to `kw2xrf_mask_irq_b'
RIOT/drivers/kw2xrf/kw2xrf_getset.c:173: undefined reference to `kw2xrf_enable_irq_b'
RIOT/examples/gnrc_networking/bin/pba-d-01-kw2x/kw2xrf.a(kw2xrf_getset.o): In function `kw2xrf_set_idle_sequence':
RIOT/drivers/kw2xrf/kw2xrf_getset.c:182: undefined reference to `kw2xrf_mask_irq_b'
RIOT/drivers/kw2xrf/kw2xrf_getset.c:218: undefined reference to `kw2xrf_enable_irq_b'
RIOT/examples/gnrc_networking/bin/pba-d-01-kw2x/kw2xrf.a(kw2xrf_getset.o): In function `kw2xrf_set_option':
RIOT/drivers/kw2xrf/kw2xrf_getset.c:386: undefined reference to `kw2xrf_clear_dreg_bit'
RIOT/drivers/kw2xrf/kw2xrf_getset.c:435: undefined reference to `kw2xrf_clear_dreg_bit'
RIOT/drivers/kw2xrf/kw2xrf_getset.c:439: undefined reference to `kw2xrf_set_dreg_bit'
RIOT/drivers/kw2xrf/kw2xrf_getset.c:454: undefined reference to `kw2xrf_clear_dreg_bit'
RIOT/drivers/kw2xrf/kw2xrf_getset.c:469: undefined reference to `kw2xrf_set_dreg_bit'
RIOT/examples/gnrc_networking/bin/pba-d-01-kw2x/kw2xrf.a(kw2xrf_intern.o): In function `kw2xrf_timer_load':
RIOT/drivers/kw2xrf/kw2xrf_intern.c:116: undefined reference to `kw2xrf_set_dreg_bit'
RIOT/examples/gnrc_networking/bin/pba-d-01-kw2x/kw2xrf.a(kw2xrf_intern.o): In function `kw2xrf_seq_timeout_on':                                                                                                                                                                         
RIOT/drivers/kw2xrf/kw2xrf_intern.c:197: undefined reference to `kw2xrf_mask_irq_b'                                                                                                                                                                                                     
RIOT/drivers/kw2xrf/kw2xrf_intern.c:206: undefined reference to `kw2xrf_set_dreg_bit'                                                                                                                                                                                                   
RIOT/drivers/kw2xrf/kw2xrf_intern.c:207: undefined reference to `kw2xrf_enable_irq_b'                                                                                                                                                                                                   
RIOT/examples/gnrc_networking/bin/pba-d-01-kw2x/kw2xrf.a(kw2xrf_intern.o): In function `kw2xrf_seq_timeout_off':                                                                                                                                                                        
RIOT/drivers/kw2xrf/kw2xrf_intern.c:212: undefined reference to `kw2xrf_clear_dreg_bit'                                                                                                                                                                                                 
collect2: error: ld returned 1 exit status                                                                                                                                                                                                                                                        
make: *** [RIOT/examples/gnrc_networking/../../Makefile.include:298: all] Error 1
```

--- a/drivers/kw2xrf/include/kw2xrf_intern.h
+++ b/drivers/kw2xrf/include/kw2xrf_intern.h
@@ -43,7 +43,7 @@ typedef enum {
  * @param[in] reg       address of register
  * @param[in] bit       set this bit
  */
-inline void kw2xrf_set_dreg_bit(kw2xrf_t *dev, uint8_t reg, uint8_t bit)
+static inline void kw2xrf_set_dreg_bit(kw2xrf_t *dev, uint8_t reg, uint8_t bit)
 {
     uint8_t tmp = kw2xrf_read_dreg(dev, reg);
     tmp |= bit;
@@ -57,7 +57,7 @@ inline void kw2xrf_set_dreg_bit(kw2xrf_t *dev, uint8_t reg, uint8_t bit)
  * @param[in] reg       address of register
  * @param[in] bit       set this bit
  */
-inline void kw2xrf_clear_dreg_bit(kw2xrf_t *dev, uint8_t reg, uint8_t bit)
+static inline void kw2xrf_clear_dreg_bit(kw2xrf_t *dev, uint8_t reg, uint8_t bit)
 {
     uint8_t tmp = kw2xrf_read_dreg(dev, reg);
     tmp &= ~bit;
@@ -69,7 +69,7 @@ inline void kw2xrf_clear_dreg_bit(kw2xrf_t *dev, uint8_t reg, uint8_t bit)
  *
  * @param[in] dev       kw2xrf device descriptor
  */
-inline void kw2xrf_enable_irq_b(kw2xrf_t *dev)
+static inline void kw2xrf_enable_irq_b(kw2xrf_t *dev)
 {
     kw2xrf_clear_dreg_bit(dev, MKW2XDM_PHY_CTRL4, MKW2XDM_PHY_CTRL4_TRCV_MSK);
 }
@@ -79,7 +79,7 @@ inline void kw2xrf_enable_irq_b(kw2xrf_t *dev)
  *
  * @param[in] dev       kw2xrf device descriptor
  */
-inline void kw2xrf_mask_irq_b(kw2xrf_t *dev)
+static inline void kw2xrf_mask_irq_b(kw2xrf_t *dev)
 {
     kw2xrf_set_dreg_bit(dev, MKW2XDM_PHY_CTRL4, MKW2XDM_PHY_CTRL4_TRCV_MSK);
 }

